cmd/gossamer/config.go

```diff
@@ -26,20 +24,24 @@ import (
 
 var (
 	// DefaultCfg is the default configuration for the node.
-	DefaultCfg                = dot.GssmrConfig
-	defaultGssmrConfigPath    = "./chain/gssmr/config.toml"
-	defaultKusamaConfigPath   = "./chain/kusama/config.toml"
-	defaultPolkadotConfigPath = "./chain/polkadot/config.toml"
-	defaultDevConfigPath      = "./chain/dev/config.toml"
-	defaultWestendConfigPath  = "./chain/westend/config.toml"
+	DefaultCfg                  = dot.PolkadotConfig
+	defaultKusamaConfigPath     = "./chain/kusama/config.toml"
+	defaultPolkadotConfigPath   = "./chain/polkadot/config.toml"
+	defaultWestendDevConfigPath = "./chain/westend-dev/config.toml"
+)
+
+const (
+	kusamaName     = "kusama"
+	polkadotName   = "polkadot"
+	westendDevName = "westend-dev"
```

Comment:
I put these as inline case statements in my westend PR since we don't reference them more than once.

Reply:
okay sounds good ill update to do that